core/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -65,6 +65,7 @@ grant {
   permission java.util.PropertyPermission "solr.directoryFactory", "write";
 
   // set by ESTestCase to improve test reproducibility
+  permission java.util.PropertyPermission "tests.seed", "write";
```

Comment:
see my other note. this is set by the build system and should not be changed. securityexception if you try to change it is correct.